examples/notebooks/datastores/6_geotiff_cog.ipynb

```diff
@@ -2184,6 +2155,172 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "level = ml_s3_dataset.get_dataset(2)\n",
+    "level.band_2.plot.imshow()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "047f873b-7a0b-470f-89de-cadad24cf6ea",
+   "metadata": {},
+   "source": [
+    "#Test accessing a aws private s3"
```

Comment:
Please add a short explanation what a user can expect if they do not have access to our testing bucket. This is an example notebook and people outside the company will get errors here.